.eslintrc.js

```diff
@@ -17,6 +21,8 @@ module.exports = {
   rules: {
     'ember/new-module-imports': 'off',
     'ember/no-restricted-resolver-tests': 'off',
+    'ember/no-test-import-export': 'off',
+    'ember/no-ember-testing-in-module-scope': 'off'
```

Comment:
We should not turn this rule off. The thing it is flagging is correct. `Ember.testing` should never be set during module evaluation.